Update the integration for compatibility with the recent Neptune client (#23)
neptune-ai/neptune-r: README.md

                    api_token="NEPTUNE_API_TOKEN")
```

> This code creates a Run in project of your choice. This will be your gateway to log metadata to Neptune. 

Reviewer: This does not make the sentence grammatically more correct. Would propose the following:

```suggestion
This code creates a run in the specified project. This will be your gateway to logging metadata to Neptune.
```

run["train/epoch/loss"]$log(loss)
```

> You can log training metrics to Neptune using series fields. In Neptune, there are three types of series - float series, string series, and file series. Each `neptune_log()` will add a new value at the end of the series. 

Reviewer: We should update instances of `log` in the docs as well.

```suggestion
You can log training metrics to Neptune using series fields. In Neptune, there are three types of series - float series, string series, and file series. Each `neptune_append()` will add a new value at the end of the series. 
```
